varats-core/varats/provider/bug/bug_provider.py

```diff
@@ -45,9 +45,15 @@ def find_all_pygit_bugs(self) -> tp.FrozenSet[bug.PygitBug]:
         Returns:
             A set of PygitBugs.
         """
+        resulting_bugs: tp.Set[bug.PygitBug] = set()
         if self.__github_project_name:
-            return bug.find_all_pygit_bugs(self.__github_project_name)
-        return frozenset()
+            resulting_bugs.union(
+                bug.find_all_issue_pygit_bugs(self.__github_project_name)
+            )
+        resulting_bugs.union(
+            bug.find_all_commit_message_pygit_bugs(self.project.name)
```

Comment:
Technically, the instances of `tp.Type[Project]` have no attribute `name`. Use `NAME` instead.
```suggestion
            bug.find_all_commit_message_pygit_bugs(self.project.NAME)
```